src/ceph-volume/ceph_volume/devices/lvm/zap.py

```diff
@@ -59,27 +60,37 @@ def zap(self, args):
         logger.info("Zapping: %s", path)
         terminal.write("Zapping: %s" % path)
 
-        if args.destroy and not lv:
-            # check if there was a pv created with the
-            # name of device
-            pv = api.PVolumes().get(pv_name=device)
-            if pv:
-                logger.info("Found a physical volume created from %s, will destroy all it's vgs and lvs", device)
-                vg_name = pv.vg_name
-                logger.info("Destroying volume group %s because --destroy was given", vg_name)
-                terminal.write("Destroying volume group %s because --destroy was given" % vg_name)
-                api.remove_vg(vg_name)
-                logger.info("Destroying physical volume %s because --destroy was given", device)
-                terminal.write("Destroying physical volume %s because --destroy was given" % device)
-                api.remove_pv(device)
-            else:
-                logger.info("Skipping --destroy because no associated physical volumes are found for %s", device)
-                terminal.write("Skipping --destroy because no associated physical volumes are found for %s" % device)
+        # check if there was a pv created with the
+        # name of device
+        pv = api.PVolumes().get(pv_name=device)
```

Comment:
this is what `get_pv()` does, any reason why using the class like that instead of the helper?